nixos/modules/tasks/network-interfaces.nix

```diff
@@ -1385,14 +1385,6 @@ in
           val = tempaddrValues.${opt}.sysctl;
          in nameValuePair "net.ipv6.conf.${replaceStrings ["."] ["/"] i.name}.use_tempaddr" val));
 
-    security.wrappers = {
-      ping = {
-        owner = "root";
-        group = "root";
-        capabilities = "cap_net_raw+p";
-        source = "${pkgs.iputils.out}/bin/ping";
-      };
-    };
     security.apparmor.policies."bin.ping".profile = lib.mkIf config.security.apparmor.policies."bin.ping".enable (lib.mkAfter ''
```

Comment:
Ah, this should be removed as well

Reply:
Thanks.  However, I would like to first learn `apparmor` itself and its NixOS module in a few days.

Reply:
done